ArmPkg/Include/Guid/SmmuConfig.h

```diff
@@ -0,0 +1,45 @@
+/** @file
+    File for SMMU config structures.
+
+    Copyright (C) Microsoft Corporation. All rights reserved.
+    SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _SMMU_CONFIG_GUID_H_
+#define _SMMU_CONFIG_GUID_H_
+
+#include <IndustryStandard/IoRemappingTable.h>
+
+typedef struct _SBSA_EFI_ACPI_6_0_IO_REMAPPING_ITS_NODE {

```

Comment:
These structs are not referenced anywhere else, and I have created them for use to create a IORT. Renamed the structs to be prefixed with "PLATFORM" generic name instead of SBSA, as the platform would configure all of this.